ocaml/default.nix

```diff
@@ -217,6 +205,14 @@ with oself;
     then oself.dune_2
     else osuper.dune_1;
 
+  dune_2 = osuper.dune_2.overrideAttrs (_: {
+    src = builtins.fetchurl {
+      url = "https://github.com/ocaml/dune/releases/download/3.0.0/fiber-3.0.0.tbz";
```

Comment:
Why fiber?

Reply:
took the file from https://github.com/ocaml/dune/releases/tag/3.0.0